install_config/configuring_authentication.adoc

```diff
@@ -1391,3 +1391,47 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
+
+Administrators can prevent clients from accessing the API by their `User-Agent`
+header in the new `*userAgentMatching*` configuration setting.
+
+For example, you can deny clients using particular libraries or binaries. This
+example denies the Kubernetes 1.2 client binary, OpenShift Origin 1.1.3
+binary, and the POST and PUT *httpVerbs*:
+
+====
+----
+policyConfig:
+  userAgentMatchingConfig:
+    defaultRejectionMessage: "Your client is too old.  Go to https://example.org to update it."
+    deniedClients:
+    - regex: '\w+/v(?:(?:1\.1\.1)|(?:1\.0\.1)) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v(?:1\.1\.3) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v1\.2\.0 \(.+/.+\) kubernetes/\w{7}'amp`
+    httpVerbs:
+    - POST
+    - PUT
+    requiredClients: null
+----
+====
+
+Administrators can also deny clients that do not exactly match the expect
+clients:
+
+====
+----
+policyConfig:
+  userAgentMatchingConfig:
+    defaultRejectionMessage: "Your client is too old.  Go to https://example.org to update it."
+    deniedClients: []
+    requiredClients:
```

Comment:
For usage of "requiredClients", better to add a note for it, otherwise, "curl" and web console cannot access the server API. Could refer to https://bugzilla.redhat.com/show_bug.cgi?id=1306590#c22 (one raised confusion) and https://bugzilla.redhat.com/show_bug.cgi?id=1306590#c23 (the answer to the confusion) and the Setup in TCMS case 520861 (the concrete demo example of the answer)